types/src/transaction/mod.rs

```diff
@@ -283,6 +283,63 @@ impl RawTransaction {
         ))
     }
 
+    /// Signs the given fee-payer `RawTransaction`, which is a transaction with secondary
+    /// signers and a gas payer in addition to a sender. The private keys of the sender, the
+    /// secondary signers, and gas payer signer are used to sign the transaction.
+    ///
+    /// The order and length of the secondary keys provided here have to match the order and
+    /// length of the `secondary_signers`.
+    pub fn sign_fee_payer(
+        self,
+        sender_private_key: &Ed25519PrivateKey,
+        secondary_signers: Vec<AccountAddress>,
+        secondary_private_keys: Vec<&Ed25519PrivateKey>,
+        fee_payer_address: AccountAddress,
+        fee_payer_private_key: &Ed25519PrivateKey,
+    ) -> Result<SignatureCheckedTransaction> {
```

Comment:
You know, eventually we'll probably want to make these signatures stacked together later, rather than have all the private keys in one place (since they should never all be in the same place)

Reply:
What do you mean by "stacked together"?